core/src/Kokkos_NumericTraits.hpp

```diff
@@ -646,24 +646,15 @@ struct reduction_identity<double> {
   KOKKOS_FORCEINLINE_FUNCTION constexpr static double min() { return DBL_MAX; }
 };
 
-#if !defined(KOKKOS_ACTIVE_EXECUTION_MEMORY_SPACE_CUDA) && \
-    !defined(KOKKOS_ACTIVE_EXECUTION_MEMORY_SPACE_HIP_GPU)
+// No __host__ __device__ annotation because long double treated as double in
+// device code.  May be revisited later if that is not true any more.
```

Comment:
```suggestion
// FIXME No __host__ __device__ annotation because long double treated as double in
// device code.  May be revisited later if that is not true any more.
```

Reply:
This is not a FIXME

Reply:
I think comments such as "Maybe revisited later" are entirely ignored if they don't have a FIXME with them.

Reply:
FIXME_MAYBE if one gpu backend supports this one day and we really need it